docs/KafkaFile.md

```diff
@@ -0,0 +1,34 @@
+# Kafka file
+
+A Kafka file is a file with `.kafka` file extension which provides the capability to declare:
+
+ * [PRODUCER](#PRODUCER) to produce records.
+ * [CONSUMER](#CONSUMER) to start /stop consumer. 
+ 
+for a given cluster. 
+
+## Select cluster
+
+Before using a `.kafka` file you must select the cluster:

```

Comment:
Before performing actions from a `.kafka` file, a cluster must be selected:

Reply:
fixed